src/SLW.js

```diff
@@ -188,6 +188,11 @@ export default class SLW {
       
       ctx.drawImage(Text.write(str), 4, 4)
     }
+
+    if (this.level.debug) {
+      ctx.drawImage(Text.write('x: '+this.player.x), 2, 2)
+      ctx.drawImage(Text.write('y: '+this.player.y), 2, 4)
```

Comment:
These are gonna appear right next to eachother BTW.
The positions are absolute.

_Also it looks like you aren't testing your code locally before pushing?_